### DISCUSSION TITLE
basn_mask field in  hires input file

### DISCUSSION THREAD

**Discussion by bfkg** [NCAR/wrf_hydro_nwm_public#638] [Development]
The high resolution input file contains a basn_mask field. As far as I understand the code, the field is never used in the model though. Instead the coarse grid basin text file is imported and assigned also to the high resolution grid. Wouldn't it be better to use the basn_mask field and to derive the coarse grid field from it? The basin text file is very much prone to errors since it is not documented what's the storage order of these numbers. Are there any thoughts about this? I would vote for getting rid of the text file and to use the high resolution field from the netcdf file.

**Comment by weiyu11** [comment]
I have an impression that we used it before. Need to confirm with Dave 
for the details. Please keep it so far. We will take a look later when 
we have time.

Thanks.

Wei

**Comment by weiyu11** [comment]
Thanks We8.  We can check this after the current NWM v1.1 deliverable.

Dave
